### PR TITLE
[10.0][MIG] account_voucher

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
This PR adds the migration scripts to v10 for account_voucher module.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:  

Good Job!

**Review by AaronHForgeFlow — APPROVED** [review]
Well done :1st_place_medal:

File: odoo/openupgrade/doc/source/modules90-100.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ missing in the new release are marked with |del|.
 +-----------------------------------+-----------------------------------+
 |account_test                       | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|account_voucher                    |                                   |
+|account_voucher                    | Done                                  |
```
Comment (cubells) [inline odoo/openupgrade/doc/source/modules90-100.rst]: Typo: there are 4 extra spaces... :)

**Review by cubells — CHANGES_REQUESTED** [review]
A typo

File: addons/account_voucher/migrations/10.0.1.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    """
+    cr.execute(
+        """
+        UPDATE account_voucher
+        LEFT JOIN account_move ON account_move.id = account_voucher.move_id
+        SET account_voucher.account_date = account_move.date
+        """)
+
+
+@openupgrade.migrate(use_env=True)
```
Comment (cubells) [inline addons/account_voucher/migrations/10.0.1.0/post-migration.py]: use_env=True is default so it's not necessary but this not block pr.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

**Review by astirpe — APPROVED** [review]

**Review by NL66278 — CHANGES_REQUESTED** [review]
This contains invalid SQL syntax.

Tip: to check SQL statements, just past them in the psql console (connected to a database migrated to 10.0).

**Comment by MiquelRForgeFlow** [comment]
@NL66278 comment attended to, thanks!! My mind thinks in MySQL instead of PostgreSQL 😅

**Pull request merged**